### DISCUSSION TITLE
stress tensor from MD simulation using CHGNET

### DISCUSSION THREAD

**Discussion by gaoxuew** [CederGroupHub/chgnet#110] [Q&A]
Dear developers,
I am interested in calculating viscosity using off-diagonal elements of stress tensor from MD simulations using CHGNET. Could you please comment how good are the off-diagonal elements of stress tensors from CHGNET? Is the ideal gas correction included in the output?
Thank you.

**Comment by bowen-bd** [comment]
I have not tested the off-diagonal parts of stress tensor explicitly, but here is my comment, I hope it's helpful.

 ` chgnet.predict_structure` does not include the ideal gas correction. But when running the MD in ASE, the ideal gass correction is added [here](https://gitlab.com/ase/ase/-/blame/master/ase/md/nptberendsen.py#L148), which is calculated from the momentum carried in ASE `atoms`.

ASE MD also assume the stress tensor is symmetric by default. 

If you have any benchmark result on the off-diagonal term, please share with us as it will be very helpful.
